Opserver/Views/SQL/Databases.Modal.BlitzCache.cshtml

```diff
@@ -0,0 +1,164 @@
+                    <td class="">
+                        @(i.QueryText?.TruncateWithEllipsis(50))
+                    </td>
+                </tr>
+                <tr class="tablesorter-childRow hidden text-wrap">
+                    <td colspan="13">
+                        <div class="panel panel-default">
+                            <div class="panel-body">
+                                <table class="table-striped col-xs-12">
+                                    <thead>
+                                        <tr>
+                                            <th>Query Ids</th>
+                                            <th>Plan Ids</th>
+                                            <th>Warnings</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        <tr>
+                                            <td>@(String.Join(",", i.QueryIds.Select(p => p.ToString()).ToArray())) </td>
+                                            <td>@(String.Join(",", i.PlanIds.Select(p => p.ToString()).ToArray())) </td>
+                                            <td>@i.Warnings</td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </div>
+                        </div>
+                        <div class="panel panel-default">
+                            <div class="panel-body">
+                                @(Html.Raw(i.QueryText?.Replace("      ", "<br>")))
+                            </div>
+                        </div>
+                    </td>
+                </tr>
+
+                <tr class="tablesorter-childRow hidden"></tr>
+            }
+        </tbody>
+        <tfoot>
+            @if (!result.Data.Any())
+            {
+                <tr>
+                    <td colspan="8">
+                        <div class="no-content">No blitz cache results</div>
+                    </td>
+                </tr>
+            }
+            else
+            {
+                <tr class="total-row">
+                    <td colspan="8">
+                        <b>Total</b> <span class="text-muted">(@result.Data.Count.Pluralize("result"))</span>
+                    </td>
+                </tr>
+            }
+        </tfoot>
+    </table>
+    <script>
+        $(function () {
+            $('.js-database-indexes').tablesorter({
+                headers: {
+                    3: { sorter: 'dataVal', sortInitialOrder: 'desc' },
+                    4: { sorter: 'dataVal', sortInitialOrder: 'desc' },
+                    5: { sorter: 'dataVal', sortInitialOrder: 'desc' },
+                    6: { sorter: 'dataVal', sortInitialOrder: 'desc' },
+                    7: { sorter: 'dataVal', sortInitialOrder: 'desc' },
+                    8: { sorter: 'dataVal', sortInitialOrder: 'desc' }
```

Comment:
I'm not familiar with the source code... but is this just overriding the initial order, or specifying what columns can be sorted?

By my reckoning I would imagine at least being able to sort by:

                <th>Cost</th>
                <th>Created</th>
                <th>Last Executed</th>
                <th>No. Executes</th>
                <th>Executions/min</th>
                <th>Total CPU</th>
                <th>Total Reads</th>
                <th>Total Duration</th>

Would be helpful - which is more than the 6 columns specified above.

Reply:
Its copy/pasta seems to just override initial order. I'll set it correctly for numerics.